users/admin.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from .forms import CustomUserCreationForm, CustomUserChangeForm
+from .models import User
+
+
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
+    add_form = CustomUserCreationForm
+    form = CustomUserChangeForm
+    list_display = ['id', 'email', 'username', 'first_name', 'last_name']
```

Comment:
is this formatted with black? We don't have the pre-commit hook in phishtray so you might have to set it up on PyCharm or whatever IDE you use! https://black.readthedocs.io/en/stable/editor_integration.html

Reply:
Probably!! Thanks